vnext/Shared/Networking/WinRTHttpResource.cpp

```diff
@@ -146,9 +155,28 @@ IAsyncOperation<HttpRequestMessage> WinRTHttpResource::CreateRequest(
       auto file = co_await StorageFile::GetFileFromApplicationUriAsync(Uri{to_hstring(data["uri"].asString())});
       auto stream = co_await file.OpenReadAsync();
       content = HttpStreamContent{std::move(stream)};
-    } else if (!data["form"].empty()) {
-      // #9535 - HTTP form data support
-      // winrt::Windows::Web::Http::HttpMultipartFormDataContent()
+    } else if (!data["formData"].empty()) {
+      winrt::Windows::Web::Http::HttpMultipartFormDataContent multiPartContent;
+      auto formData = data["formData"];
+
+      for (auto &formDataPart : formData) {
+        IHttpContent formContent{nullptr};
+        if (!formDataPart["string"].empty()) {
+          formContent = HttpStringContent{to_hstring(formDataPart["string"].asString())};
+        } else if (!formDataPart["uri"].empty()) {
```

Comment:
>formDataPart["uri"] [](http://example.com/codeflow?start=20&length=19)

are formDataPart["string"] and formDataPart["uri"] mutually exclusive?

Reply:
Yes. A single HTTP request payload can be of types:
- `string`
- `base64` (binary)
- `uri` (file content reference)
- `formData`
  - Arbitrary sequence of `string`, `base64`, or `uri`.